src/libsync/propagatedownload.cpp

```diff
@@ -563,7 +563,6 @@ void PropagateDownloadFile::startAfterIsEncryptedIsChecked()
         return checksum_header.startsWith("SHA")
             || checksum_header.startsWith("MD5:");
     };
-    Q_ASSERT(_item->_modtime > 0);
```

Comment:
@mgallien Why remove this?

Reply:
this makes it impossible to sync against a server with files that have a `modtime` equal to `0`
in that case the assert will fire and crash the application
the code is indeed able to deal with that situation now so the assert is not needed